src/organisms/documentation/general.js

```diff
@@ -35,20 +39,20 @@ const propsByApp = {
   cloud: {
     documentationUrl: "https://learn.netdata.cloud/",
     issuesUrl: "https://github.com/netdata/netdata-cloud/issues/new",
-    issuesLabel: "Raise a bug about Netdata Cloud",
+    issuesLabel: "Raise a bug about Netdata Cloud or inform us for any issue you face.",
     otherIssuesUrl: "https://github.com/netdata/netdata/issues/new",
-    otherIssuesLabel: "an Agent",
+    otherIssuesLabel: "Agent",
     communityUrl: "https://community.netdata.cloud/c/support/13",
-    communitySupportUrl: "https://community.netdata.cloud/c/support/13/none",
+    communitySupportUrl: " https://www.netdata.cloud/community/",
```

Comment:
There is an extra space here

Reply:
Oops! I thought I pushed those typo fixes!